REVIEWING.md

```diff
@@ -0,0 +1,32 @@
+# Reviewing contributed session
+
+## Activities / Challenges
+
+You will make sure that activities / challenges are of good quality:
```

Comment:
```suggestion
Make sure that activities / challenges are of good quality:
```
Let's keep sentencens imperative as opposed to future. They are more compelling like this.